File: p7/p7.jl
function is_prime(n)
	if n%2==0
		return false
	end
	for i = 3:2:floor(sqrt(n))+1
		if n%i==0
			return false
		end	
	end
	return true
end	



function nthprime(nprime)
	pnum = 2
	p = 3
	while pnum < nprime
		p+=2
		if is_prime(p)
			pnum += 1	
		end
	end	
	return p
end



function answer()
	return nthprime(10001)
end	

File: p7/p7.py
import numpy as np


def is_prime(n):
    if n%2==0:
        return False
    for i in range(3, int(np.sqrt(n))+1, 2):
        if n%i==0:
            return False
    return True


def nthprime(primeN):
    pnum = 2
    p = 3
    while pnum < primeN:
        p += 2 
        if is_prime(p):
            pnum += 1
    return p

def answer():
    return nthprime(10001)


